src/ui/Text.js

```diff
@@ -1,9 +1,23 @@
 import PropTypes from 'prop-types';
```

Comment:
I don't know for sure if this requires a new variant for `Text` since this code can be reduced to 
`<Text color={getValue('muted.color')} />`.

Reply:
Then we have to do that everywhere where we want to have muted text?

Reply:
Plus it opens up a lot of flexibility, instead of making sure that text is always one of two colors every color becomes possible?

Reply:
Yes if muted text is used a lot a variant can be handy. But you should use the color prop over the css props, it is exposed by Box